style.css

```diff
@@ -0,0 +1,143 @@
+*{
+    padding:0;
+    margin:0;
+    box-sizing: border-box;
+}
+html{
+    height: 100%; 
+    width: 100%;
+}
+body{
+    background-color: white;
+    margin: 0;
+    padding: 0;
+    border: 0;
+    height: 100%; 
+    width: 100%;
+    max-height: 100%; 
+}
+
+
+/******************************************************************/
+/*FOR THE MAIN-FLEX*/
+.main-container{
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+}
+
+/******************************************************************/
+/*FOR THE HEADER (NAV BAR)*/
+header{
+    box-sizing: border-box;
+    background-color: #171515;
+    color: white;
+    flex: 80px 0 0; /*80px height, don't grow or shrink*/
+    padding: 1.3em 1.5em;
+    display: flex;
+    justify-content: space-between;
+}
```

Comment:
please add space between each 2 classes

Reply:
to be honest I didn't really understand what do you mean by 'space', there is already a space-between the header elements

Reply:
done